README.md

```diff
@@ -9,6 +9,7 @@ A modern, production-ready template for building full-stack React applications u
 - 📦 Modern bundling with Vite
 - 🔄 Data loading and mutations with React Router
 - 🔍 tRPC + React Query with server-side prefetching support
+- ⚙️ React Server Components with client/server comparison examples
```

Comment:
can we just say rsc why client/server comparison examples

Reply:
you didn't really address it. I want it to not say "client/server comparison examples at all". maybe just "React Server Components"